qlib/utils/paral.py

```
@@ -88,32 +88,28 @@ def __init__(self) -> None:
             except Empty:
-                # NOTE: avoid deadlock. make checking main thread possible
                 continue
             if data == self.STOP_MARK:
                 break
             data()
 
     def __call__(self, func, *args, **kwargs):
-        self._q.put(partial(func, *args, **kwargs))
+        func(*args, **kwargs)
 
     def wait(self, close=True):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)